packages/pwa-kit-dev/src/configs/webpack/config.js

```diff
@@ -145,6 +145,29 @@ const findDepInStack = (pkg) => {
     return candidate
 }
 
+// Helper function to detect extensions
+const detectExtensions = () => {
```

Comment:
it maybe more easily testable to have the dependencies listed in the function args?

```suggestion
const detectExtensions = ({projectDir}) => {
```